### DISCUSSION TITLE
My college doesn't make english variants of studend id

### DISCUSSION THREAD

**Discussion by oleshkus** [community/community#138652] [GitHub Education] [answered]
### Hi everyone,

What should i do if my college doesn't make english copies of documents? I have "student ticket" - an analogue of the student ID. Can it be accepted without english variant?

**Comment by alx-mp4** [comment, marked as answer]
Hi @oleshkus, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!

**Comment by oleshkus** [comment]
The solution was just translating document manually. Main things to translate - first name, last name, expiration date of your student status. It helped me to approve my appeal. Want to add that im from Belarus and I was using my student ticket to approve.

**Reply by alx-mp4** [reply]
Glad that helped!

**Reply by me-when-the-uh** [reply]
Same for me, scanning the student ID and my student status certificate and providing a translation of the important parts of the documents below the documents has helped. I used MS Paint to type the translation and the transcription of my name in both languages. Getting the Student Pack benefits was easier 2 years ago, nowadays you have to make sure the text is crystal clear and machine readable, even if your university has a partnership with Microsoft.
